src/settings.ts

```diff
@@ -349,6 +320,11 @@ export const BURN_VENUE_TEMPLATES_V2: Array<Template_v2> = [
       "Insert almost anything into a styled iFrame. This space does not have video chatting.",
     ],
   },
+  {
+    template: VenueTemplate.screeningroom,
+    name: "Screening room",
+    description: ["Add an screening room with the videos listed inside."],
+  },
```

Comment:
why was this added here? it's not stated in the issue

Reply:
got it. It was fixed in ddb4153701dddc332fba7b62305e4b95c1da6ccc